### PR TITLE
fix(config): add missing parameters for Telldus TZWP-102

### PR DESCRIPTION
**Pull request by norrland**
Enable additional reporting configurations.

* 17 - Report on power consumption change %
* 101 - Power (W) report interval
* 102 - Energy (kWh) report interval
* 103 - Voltage (V) report interval
* 104 - Current (A) report interval

### CODE REVIEW

File: packages/config/config/devices/0x0176/tzwp-102.json [hunk 1]
```diff
@@ -93,6 +93,76 @@
...
+				{
+					"label": "Disabled",
+					"value": 0
+				}
+			]
+		},
+		{
+			"#": "101",
+			"label": "Power (W) report interval",
+			"valueSize": 4,
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0176/tzwp-102.json:117]:
```suggestion
			"valueSize": 4,
			"unit": "seconds",
```
Do the same for the following parameters please.

File: packages/config/config/devices/0x0176/tzwp-102.json [hunk 2]
```diff
@@ -93,6 +93,76 @@
 					"value": 0
 				}
 			]
+		},
+		{
+			"#": "17",
+			"label": "Report on change of power consumption (%)",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0176/tzwp-102.json]: We should distingish between param 16 and this one better. I suggest:
Param 16:
- change label to "Power Consumption Reports: Change Threshold"
- add unit `W`

Param 17:
- change label to "Power Consumption Reports: Relative Change Threshold"
- add unit `%`

**Review by AlCalzone — CHANGES_REQUESTED** [review]

File: packages/config/config/devices/0x0176/tzwp-102.json [hunk 3]
```diff
@@ -93,6 +94,81 @@
...
+					"value": 0
+				}
+			]
+		},
+		{
+			"#": "101",
+			"label": "Power (W) report interval",
+			"valueSize": 4,
+			"unit": "seconds",
+			"minValue": 5,
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0176/tzwp-102.json]: Oh... one more thing I missed. `minValue` needs to be 0 if there is an option for 0. We generally explain the gap in the valid range with a description, e.g. "Allowable range: 5-2678400"

Same for the following parameters

**Review by AlCalzone — CHANGES_REQUESTED** [review]

File: packages/config/config/devices/0x0176/tzwp-102.json (hunk 3 above)
Comment (norrland) [reply packages/config/config/devices/0x0176/tzwp-102.json]: Gotcha, was wondering about that one. If it accepted an array of [0, 5-n] or something.
Fixing coming up.

**Review by AlCalzone — APPROVED** [review]

**Comment by AlCalzone** [comment]
Thanks!

**Pull request merged**